### PR TITLE
RHDEVDOCS-5591: Content changes as per SME feedback

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.13

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5591

Link to docs preview:

https://75384--ocpdocs-pr.netlify.app/openshift-gitops/latest/accesscontrol_usermanagement/configuring-argo-cd-rbac.html

SME review: @svghadi 
QE review: @varshab1210
Internal Peer review: @eromanova97
Peer review:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc [hunk 1]
```diff
@@ -4,9 +4,14 @@ include::_attributes/common-attributes.adoc[]
 = Configuring Argo CD RBAC
 :context: configuring-argo-cd-rbac
 
-toc::[]
+By default, any type of user except the `kube:admin` user, logged into Argo CD does not have access to any services by default.
```
Comment (svghadi) [inline accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc:7]:
```suggestion
By default, any type of user except the `kube:admin` user, logged into default `openshift-gitops` Argo CD instance does not have access to any services by default.
```

**Review by svghadi — CHANGES_REQUESTED** [review]

File: accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc:7]: Hi @svghadi - I would refrain from using `openshift-gitops` explicitly as the user is aware of the disitinction between a default and a custom Argo CD instance.

Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc:7]: I have included the remaining part of your feedback in the content edit.

**Review by svghadi — APPROVED** [review]
/lgtm
Thanks @Dhruv-Soni11

**Comment by varshab1210** [comment]
LGTM, thanks

File: accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc [hunk 2]
```diff
@@ -5,8 +5,12 @@ include::_attributes/common-attributes.adoc[]
 :context: configuring-argo-cd-rbac
 
 toc::[]
+By default, any type of user except the `kube:admin` user, logged into the default Argo CD instance does not have access to any services. But a user logged into a custom Argo CD instance is a read-only user by default.
 
-By default, if you are logged in to Argo CD using Red Hat SSO (RH SSO), you are a read-only user. You can change and manage the user level access.
+[NOTE]
+====
+* In {gitops-title} v1.9.0 or earlier versions, any type of user, except the `kube:admin` user, logged into Argo CD using Red Hat SSO (RH SSO), is a read-only user by default.
```
Comment (tmalove) [inline accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc:12]:
```suggestion
* In {gitops-title} v1.9.0 or earlier versions, any type of user, except the `kube:admin` user, logged into Argo CD using Red Hat SSO (RH SSO) is a read-only user by default.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas

Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc:12]: Fixed.

**Review by kcarmichael08 — COMMENTED** [review]
Sorry, I think there are a couple very small things that should be fixed before merging.

File: accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc [hunk 3]
```diff
@@ -5,8 +5,12 @@ include::_attributes/common-attributes.adoc[]
 :context: configuring-argo-cd-rbac
 
 toc::[]
+By default, any type of user except the `kube:admin` user, logged into the default Argo CD instance does not have access to any services. But a user logged into a custom Argo CD instance is a read-only user by default.
 
-By default, if you are logged in to Argo CD using Red Hat SSO (RH SSO), you are a read-only user. You can change and manage the user level access.
+[NOTE]
+====
+* In {gitops-title} v1.9.0 or earlier versions, any type of user, except the `kube:admin` user, logged into Argo CD using Red Hat SSO (RH SSO) is a read-only user by default.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc:12]: Fixed.

File: accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc [hunk 4]
```diff
@@ -5,8 +5,12 @@ include::_attributes/common-attributes.adoc[]
 :context: configuring-argo-cd-rbac
 
 toc::[]
+By default, any type of user except the `kube:admin` user, logged into the default Argo CD instance does not have access to any services. But a user logged into a custom Argo CD instance is a read-only user by default.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc]: Fixed.

**Comment by Dhruv-Soni11** [comment]
Hi @kcarmichael08, I appreciate the last-minute suggestions before the eventual merge.  I have gone ahead and incorporated your feedback. PTAL and let me know what you think. Thank you.

**Comment by kcarmichael08** [comment]
Hi, looks like the cherry pick to 1.13 is failing - can you please [create a manual cherry pick](https://docs.google.com/document/d/e/2PACX-1vRLKWEMHQ3DZroxZKfTu3XcrSdREr6D3oSSayBanEprXhkA2Ciyr2SQuDTYI4aIKUiOPPIQMHgjHeh8/pub)? TY!

**Comment by Dhruv-Soni11** [comment]
> Hi, looks like the cherry pick to 1.13 is failing - can you please [create a manual cherry pick](https://docs.google.com/document/d/e/2PACX-1vRLKWEMHQ3DZroxZKfTu3XcrSdREr6D3oSSayBanEprXhkA2Ciyr2SQuDTYI4aIKUiOPPIQMHgjHeh8/pub)? TY!



> Hi, looks like the cherry pick to 1.13 is failing - can you please [create a manual cherry pick](https://docs.google.com/document/d/e/2PACX-1vRLKWEMHQ3DZroxZKfTu3XcrSdREr6D3oSSayBanEprXhkA2Ciyr2SQuDTYI4aIKUiOPPIQMHgjHeh8/pub)? TY!

Hi, I have created the manual cherry pick for the GitOps 1.13 in this [PR]https://github.com/openshift/openshift-docs/pull/75970). PTAL.

**Pull request merged**